.github/workflows/build.yml

```diff
@@ -0,0 +1,56 @@
+name: build
+
+on:
+  push:
+    branches:
+      - cluster
+      - master
+    paths-ignore:
+      - '**.md'
+      - 'dashboards/**'
+      - 'deployment/**.yml'
+      - 'docs/**'
```

Comment:
Can we change ignore path to files that should trigger the build?
And build should trigger if Makfefile, Dockerfile or **.go has been changed.
Please correct me if I'm wrong @zekker6 @f41gh7